packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -703,6 +705,16 @@ export const useArbTokenBridge = (
       const l1Addresses: string[] = []
       const l2Addresses: string[] = []
 
+      // USDC is not on any token list as it's unbridgeable
+      // but we still want to detect its balance on user's wallet
+      if (isNetwork(l2ChainID).isArbitrumOne) {
+        l2Addresses.push(CommonAddress.ArbitrumOne.USDC)
+      }
+      // Arb Goerli is for e2e tests
```

Comment:
Not sure why this comment is here, we want to use Arbitrum Goerli USDC to test CCTP

Reply:
deleted